src/react/portfolio/VanillaJSProjects.tsx

```diff
@@ -1,116 +1,203 @@
 import { ProjectDetails } from './ProjectDetails'
+import { id as reactAnalogClockID } from '../reactProject_AnalogSVGClock/main'
+import { id as reactLCDClockID } from '../reactProject_LCDClock/main'
+import { id as reactTODOListID } from '../reactProject_TODOList/main'
+import { id as reactHangmanGameID } from '../reactProject_Hangman/main'
+import { id as reactSimonSaysGameID } from '../reactProject_SimonSays/main'
+import { id as reactTicTacToeGameID } from '../reactProject_TicTacToe/main'
+import { id as reactRPSGameID } from '../reactProject_RPS/main'
+import { id as reactTipCalculatorID } from '../reactProject_TipCalculator/main'
 const projects = [
   {
+    id: 'vanillajs-calender',
     title: 'CALENDER',
-    description: 'Calender',
+    description: <div>
+      For those who still prefer to rely on a calendar to check the date, here&apos;s a digital calender app that highlights the
+      current date and keeps you and everyone in your family updated with number of days in each and every month of the year.
+      Most importantly, no paper is discarded in this process and you no longer require to turn or toss pages to set correct
+      date on calender. The App does it for you!
+    </div>,
     skills: ['JavaScript', 'CSS', 'HTML', 'Git', 'Github', 'VS Code'],
```

Comment:
Looking at the `description`, I like them, but I also think they do not talk much about the technical aspect of the project. Maybe you should add a new key called `techDescription` and describe it in there. You can display that below the description in a separate section in `ProjectDetails` component.

```suggestion
    </div>,
    techDescription: <div>
      All of the date manipulation calculations in this application were implemented using the built-in <code>Date</code> object
      of the browser. This makes the page very light-weight as no external packages need to be used.
    </div>,
```

I suggest doing this for the other projects too. Just try to highlight something technically interesting about them. Because otherwise, the reader does not really know what is so special about the page.